### ISSUE TITLE
Keep script from crashing (if no match was found)

### ISSUE THREAD

**Issue by 42sol-eu** [MartinPacker/md2pptx#9]
Please close if this makes no sense to you

```
    # [Issue 9](https://github.com/MartinPacker/md2pptx/issues/9)
    if matchInfo == None:
        continue 
        # > NOTE: 
        # Traceback (most recent call last):
        # File "md2pptx", line 1997, in <module>
        #    name = matchInfo.group(1).strip()
        # AttributeError: 'NoneType' object has no attribute 'group'
```

**Comment by MartinPacker** [comment]
Would like to fix. Can you give me an example of input that causes the crash? (Until then I'm not sure what the correct fix would be.) Thanks!

**Comment by 42sol-eu** [comment]
my problem was an empty input (not loading the file cause of the path issue)

I will look something up if you like. There is surely a case that produces the same effect.

**Comment by MartinPacker** [comment]
If I look at that line it seems to be parsing a metadata line. To do that the metadata line must exist - and must look like a metadata line i.e.    key:value

Can you give me an example file?

**Comment by 42sol-eu** [comment]
Fixed with the "No Input Data found" in v1.0 
(I could not get the same error results)
